include/SimCore/SDs/EcalSD.h

```diff
@@ -70,8 +97,8 @@ class EcalSD : public CalorimeterSD {
    */
   std::map<G4VSolid*, G4Polyhedron*> polyMap_;
 
-  /// ConditionsInterface
-  ConditionsInterface& conditionsIntf_;
```

Comment:
EcalSD and HcalSD need ConditionsInterface so they can access the Geometry conditions for those two subsystems.